src/Core/ClassInfo.php

```diff
@@ -400,7 +400,7 @@ public static function shortName($nameOrObject)
      */
     public static function hasMethod($object, $method)
     {
-        if (empty($object)) {
+        if (empty($object) || (!is_object($object) && !is_string($object))) {
```

Comment:
Is this only valid use case for passing in a string a class name? 

In php7 this will be `true` var_dump(method_exists('DateTime', 'setTime'));

Might be worth expanding this to `(!is_object($object) && !is_string($object) && !class_exists($object))`

Reply:
As soon as `$object` is found to be a string, it'll stop processing, so adding `!class_exists($object)` isn't going to do anything useful there. `method_exists()` will handle cases where the provided class name doesn't exist.